.github/workflows/pr-upgrade.yml

```diff
@@ -75,7 +75,9 @@ jobs:
       - name: Run vanilla go tests on latest tag
         shell: bash
         run: |
-          go test -v -timeout=10m -run=TestUpgrade/before_upgrade ./test/e2e-migrated/upgrade/...
+          go test -v -timeout=10m -run=TestLogsUpgrade/before_upgrade ./test/e2e-migrated/upgrade/...
```

Comment:
Shouldn't we run TestUpgrade before checking out the current revision and TestLogUpgrade after?